projects/packages/forms/src/modules/file-field/view.js

```diff
@@ -131,24 +129,18 @@ const updateFileContext = ( updatedFile, fileId ) => {
 /**
  * Remove file from the temporary folder.
  *
- * @param {string} token - The token of the file to remove.
+ * @param {string} fileId - The file ID to remove.
  */
-const removeFile = token => {
-	const { endpoint, wp_nonce, jp_nonce } = getConfig( NAMESPACE );
+const removeFile = fileId => {
+	const { endpoint, uploadToken } = getConfig( NAMESPACE );
+	const formData = new FormData();
+	formData.append( 'file_id', fileId );
+	formData.append( 'upload_token', uploadToken );
 
-	const request = new Request( endpoint + '/remove', {
+	fetch( `${ endpoint }/remove`, {
 		method: 'POST',
-		headers: {
-			'Content-Type': 'application/json',
-		},
-		body: JSON.stringify( {
-			token,
-			context: 'jetpack-form',
-			wp_nonce: wp_nonce,
-			jp_upload_nonce: jp_nonce,
-		} ),
-	} );
-	fetch( request );
+		body: formData,
+	} ).then( response => response.json() );
```

Comment:
Was just revisiting this PR to check for changes. I thought I left a comment here but I don't see it, so leaving now. Not sure if we want/need to add any error handling here on the fetch?

Reply:
The file gets removed or cleaned up later, so I don’t think it’s really needed. Let’s test it out and see how it behaves once it’s live!